.eslintrc.json

```diff
@@ -9,6 +9,10 @@
       "sourceType": "module"
     },
     "rules": {
-        "no-console": "off"
-    }
+      "no-console": "off",
+      "semi": ["error", "never"],
+      "no-unused-vars": "off"
```

Comment:
why?

Reply:
No actual reason. Changed it to Error :)